src/pyDataverse/api.py

```diff
@@ -1288,7 +1287,7 @@ def edit_dataset_metadata(
             url = "{0}/datasets/editMetadata/{1}".format(
                 self.base_url_api_native, identifier
             )
-        params = {"replace": True} if replace else {}
```

Comment:
This can stay as it is. Maybe with requests this was necessary, but with httpx I tried this:
```python
>>> import httpx
>>> httpx.get('https://example.org', params={'bool': True, 'string': 'true'}).url
URL('https://example.org?bool=true&string=true')
```
So both seem to result in the same outcome. One could even consider using `params = {"replace": replace}` instead of the ternary. But I am not sure if the API supports it, so I would keep it as-is.